### PR TITLE
Bringing in new versions of packages

### PR DESCRIPTION
**Pull request by dannewnsjump24**
This PR brings in the latest version of packages to allow us to utilise some of the speed improvements in PHPStan in our CakePHP app

### CODE REVIEW

**Comment by mantebalzaraviciute** [comment]
@dannewnsjump24 Whilst youre there can you also update the ecs example in the readme please? It no longer takes in Containerconfiguration.


It takes the `ECSConfig $ecsConfig` instead.

File: composer.json [hunk 1]
```diff
@@ -8,13 +8,24 @@
             "email": "chris@jump24.co.uk",
             "homepage": "https://jump24.co.uk",
             "role": "Developer"
+        },
+        {
```
Comment (AlexJump24) [inline composer.json:12]: Typo in your name 👍🏼

Comment (dannewnsjump24) [reply composer.json:12]: resolved

File: composer.json [hunk 2]
```diff
@@ -8,13 +8,24 @@
...
+            "name": "Daniel Newn",
```
Comment (mantebalzaraviciute) [inline composer.json]: lol 😆

**Review by mantebalzaraviciute — APPROVED** [review]
awesome!

**Review by mmorganJ24 — APPROVED** [review]

**Pull request merged**